### DISCUSSION TITLE
Console app ORM support

### DISCUSSION THREAD

**Discussion by atreeon** [serverpod/serverpod#224] [Q&A]
I'd like to have a console app that interacts with my serverpod database.  

Is it possible to run serverpod via a Console App and if so is there some example code?

**Comment by atreeon** [comment]
it might not be by design but this was actually quite straightforward to achieve in the end.  I just needed the password & development files to exist in the config directory.

```
  var serverpod = Serverpod([], Protocol(), Endpoints());
  var session = InternalSession(server: serverpod.server);
  var myRow = Company(name: 'Serverpod corp.');
  await Company.insert(session, myRow);
```

**Comment by vlidholt** [comment]
@atreeon, if you create a new `Session` object, you are responsible for closing it with `session.close()`. Otherwise, it's possible you will get memory leaks in your server.

**Comment by atreeon** [comment]
Thank you @vlidholt
